src/index.js

```diff
@@ -1 +1,20 @@
+import counter from "./modules/helper.js";
+import shows from "./modules/shows.js";
+import { populateView } from "./modules/render.js";
 import './index.css';
+
+
+const showCount = document.getElementById('show-count');
+const movieSection = document.getElementById('movies');
+const refresh = async () => {
```

Comment:
## Highlights
- nice and clean code 
- impressive JS syntax

Reply:
Hi @King-Benx , 
I must say you've done an excellent job.

## Hilights 
- Descriptive PR
- Clean and readable code
- Functions working
There's nothing else to say other than merge it. 🥇 

![giphy](https://user-images.githubusercontent.com/102650102/226791753-e3e7b6c3-bb47-4c0c-8630-798be68902fd.gif)